index.html

```diff
@@ -15,7 +15,7 @@ <h2>RPIP Statuses</h2>
   <li><strong>Final</strong> - This RPIP represents the final standard. A Final RPIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.</li>
   <li><strong>Stagnant</strong> - Any RPIP in Draft or Review which becomes inactive is moved to Stagnant. An RPIP may be resurrected from this state by Authors or RPIP Editors through moving it back to Draft.</li>
   <li><strong>Withdrawn</strong> - The RPIP Author(s) have withdrawn the proposed RPIP. This state has finality and can no longer be resurrected using this RPIP number. If the idea is pursued at later date, it is considered a new proposal.</li>
-  <li><strong>Living</strong> - A special status for RPIPs that are designed to be continually updated and not reach a state of finality. This includes most notably RPIP-1.</li>
+  <li><strong>Living</strong> - An RPIP that is designed to be continually updated and not reach a state of finality.</li>
```

Comment:
Not purely aesthetic/correction. This feels appropriate and isn't even in an RPIP, but would like one more rpip editor to agree with me.

Reply:
LGTM